lib/schema.js

```diff
@@ -571,7 +571,8 @@ export default class Schema {
     if (this.refs && path) {
       path = path.toLowerCase();
       for (const key in this.refs) {
-        if (this.refs.hasOwnProperty(key) && this.refs[key].path.toLowerCase() === path) {
+        let resolvedPath = this.refs[key].path.toLowerCase();
```

Comment:
I suppose we could, but I think it looks cleaner this way. Negligible difference.